src/main/resources/plugins.config

```diff
@@ -9,8 +9,9 @@
 
 # Generate the jar file using: "jar cvfM LocalThickness_.jar *"
 
-Analyze>Local Thickness, "Local Thickness (complete process)", Local_Thickness_Driver("run")
-Analyze>Local Thickness, "Geometry to Distance Map", EDT_S1D("run")
-Analyze>Local Thickness, "Distance Map to Distance Ridge", Distance_Ridge("run")
-Analyze>Local Thickness, "Distance Ridge to Local Thickness", Local_Thickness_Parallel("run")
-Analyze>Local Thickness, "Local Thickness to Cleaned-Up Local Thickness", Clean_Up_Local_Thickness("run")
+Plugins>Local Thickness, "Local Thickness (complete process)", net.imagej.localThickness.Local_Thickness_Driver
```

Comment:
The "Analyze" menu wasn't such a bad place for this plugin. Putting one more entry to the "Plugins" menu will clutter this even more: it already contains more than 30 entries in a normal Fiji installation, and likely many more when you activated additional update sites.

I would much prefer to have the plugin menu organized in submenus according to image processing techniques. (I know that this is difficult to achieve: when looking for tools related to image registration, you find them in the `Feature Extraction`, `Registration`, `Transform` submenus, but also in `Landmarks`, for example...)

Rainer Engel has recently started an effort to get things organized, but it seems to have stalled:
http://imagej.1557.x6.nabble.com/OT-tool-categories-tp5014809.html